src/printer.rs

```diff
@@ -89,6 +89,8 @@ pub struct Printer<W> {
     null: bool,
     /// Print only the matched (non-empty) parts of a matching line
     only_matching: bool,
+    /// If only_matching is true, print only the given capture number.
+    only_capture: usize,
```

Comment:
For reference, this should probably be an `Option<usize>`, where `Some(0)` is equivalent to `--only-matching`.